egs2/reverb/enh1/local/Generate_mcTrainData_cut.m

```diff
@@ -0,0 +1,246 @@
+function Generate_mcTrainData_cut(WSJ_dir_name, save_dir)
```

Comment:
Is this the same as the one in `egs/reverb/asr1/local/Generate_mcTrainData_cut.m`?

Reply:
No, I modified it to generate aligned reference signals.

Reply:
It is better if you put some notes or rename it, in case someone uses the one in asr1 recipe by mistake in the future.

Reply:
This script can be actually used for ASR, as it does not change any signal used for ASR. It just adds new signals for speech enhancement.